clients/git/client.go

```diff
@@ -201,9 +199,81 @@ func (c *Client) GetFileContent(filename string) ([]byte, error) {
 	return nil, nil
 }
 
+func (c *Client) IsArchived() (bool, error) {
+	return false, nil
+}
+
+func (c *Client) URI() string {
+	return ""
+}
+
 func (c *Client) Close() error {
 	if err := os.RemoveAll(c.tempDir); err != nil && !os.IsNotExist(err) {
 		return fmt.Errorf("os.RemoveAll: %w", err)
 	}
 	return nil
 }
+
+func (c *Client) GetBranch(branch string) (*clients.BranchRef, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) GetCreatedAt() (time.Time, error) {
+	return time.Time{}, errNotImplemented
+}
+
+func (c *Client) GetDefaultBranchName() (string, error) {
+	return "", errNotImplemented
+}
+
+func (c *Client) GetDefaultBranch() (*clients.BranchRef, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) GetOrgRepoClient(ctx context.Context) (clients.RepoClient, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListIssues() ([]clients.Issue, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListLicenses() ([]clients.License, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListReleases() ([]clients.Release, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListContributors() ([]clients.User, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListSuccessfulWorkflowRuns(filename string) ([]clients.WorkflowRun, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListCheckRunsForRef(ref string) ([]clients.CheckRun, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListStatuses(ref string) ([]clients.Status, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListWebhooks() ([]clients.Webhook, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) ListProgrammingLanguages() ([]clients.Language, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) SearchCommits(request clients.SearchCommitsOptions) ([]clients.Commit, error) {
+	return nil, errNotImplemented
+}
+
+func (c *Client) LocalPath() (string, error) {
+	return "", errNotImplemented
```

Comment:
this would be the tempdir correct? So we could implement this

Reply:
👍